sdk/storage/Azure.Storage.Files.Shares/src/ShareClient.cs

```diff
@@ -444,6 +448,374 @@ private void SetNameFieldsIfNull()
         }
         #endregion Create
 
+        #region Create If Not Exists
```

Comment:
NIT - remove spaces. Also I believe for blobs, I put the `CreateIfNotExists` methods in the `Create` region

Reply:
Removed `CreateIfNotExists` per Ted's comment.